core/src/main/java/hudson/model/RSS.java

```diff
@@ -42,25 +41,6 @@
  */
 public final class RSS {
 
-    /**
-     * Parses trackback ping.
-     */
-    public static void doTrackback( Object it, StaplerRequest req, StaplerResponse rsp ) throws IOException, ServletException {
```

Comment:
I was unable to find any usages, but it would be great to have some binary compatibility analysis when such changes are submitted. Also not sure why it needs to be renamed